tracim/tracim/config/app_cfg.py

```diff
@@ -436,6 +439,36 @@ def __init__(self):
             # ContentType.Folder -- Folder is skipped
         ]
 
+        self.JITSI_MEET_ACTIVATED = asbool(tg.config.get(
+            'jitsi_meet.activated',
+            False,
+        ))
+        self.JITSI_MEET_DOMAIN = tg.config.get(
+            'jitsi_meet.domain'
+        )
+        self.JITSI_MEET_USE_TOKEN = asbool(tg.config.get(
+            'jitsi_meet.use_token',
+            False,
+        ))
+        self.JITSI_MEET_TOKEN_GENERATOR = tg.config.get(
+            'jitsi_meet.token_generator',
+            'local'
+        )
+        self.JITSI_MEET_TOKEN_GENERATOR_LOCAL_APP_ID = tg.config.get(
+            'jitsi_meet.token_generator.local.app_id'
+        )
+        self.JITSI_MEET_TOKEN_GENERATOR_LOCAL_SECRET = tg.config.get(
+            'jitsi_meet.token_generator.local.secret'
+        )
+        self.JITSI_MEET_TOKEN_GENERATOR_LOCAL_ALG = tg.config.get(
```

Comment:
ALG is not understandable. By the way, your code is buggy because `jitsi_meet.token_generator.local.alg` does not exist in the config template (its `.algo`, not `.alg`)

TODO : rename ALG to ALGORITHM (even if it is long... its a very local variable name, so not a problem)